packages/app/obojobo-repository/shared/actions/dashboard-actions.js

```diff
@@ -279,5 +297,6 @@ module.exports = {
 	showModuleMore,
 	showVersionHistory,
 	restoreVersion,
-	importModuleFile
+	importModuleFile,
```

Comment:
I don't see this being used in the diff, just verifying it's exported & added to Dashboard props is still needed?

Reply:
Hmm, looks like it's still being passed and used in the Dashboard for uploads

https://github.com/ucfopen/Obojobo/blob/2626f5b26096987e97ce7f312b7a2da3a2dda6de/packages/app/obojobo-repository/shared/components/dashboard.jsx#L145

Reply:
oops, that's my issue. I saw green + and managed not to see it was already in the previous version.  thanks for humoring me.